### DISCUSSION TITLE
Is Django Rest Framework a Slow Framework?

### DISCUSSION THREAD

**Discussion by FazlulAyanKoushik** [community/community#127446] [Discussions]
### Select Topic Area

Question

### Body

Is Django Rest Framework (DRF) considered a slow framework? Assessing its performance in comparison to other frameworks is crucial for understanding its efficiency in handling web APIs. Insights on its speed and optimization practices would be valuable.

**Comment by MuhamedHabib** [comment]
compared to spring boot yes .. cause django is based on python which is an intrepreted language and that's slow down time response for requests
